bam_markdup.c

```diff
@@ -852,6 +853,195 @@ static int mark_duplicates(md_param_t *param, khash_t(duplicates) *dup_hash, bam
 }
 
 
+static inline int optical_retag(md_param_t *param, khash_t(duplicates) *dup_hash, bam1_t *b, int paired, long *optical_single, long *optical_pair) {
+    int ret = 0;
+    uint8_t *data;
+
+    // remove any existing do tag
```

Comment:
"do" typo of "dt"?

Reply:
Yes it is.  I use (and remove) 'do' and 'dt' tags.  Must have got them mixed up here.